src/rust/engine/fs/src/lib.rs

```diff
@@ -442,33 +442,53 @@ impl StrictGlobMatching {
   }
 }
 
+#[derive(Debug)]
+pub enum Conjunction {
```

Comment:
Ditto making this a single-use, self-documenting type rather than a generic conjunction.

Reply:
Also done in b5eec2d48732c77f7def517e49e6046a6334ad8c!